ghostfolio/umbrel-app.yml

```diff
@@ -31,19 +31,17 @@ path: ""
 deterministicPassword: false
 torOnly: false
 releaseNotes: >
-  This release updates Ghostfolio from 1.231.0 to 1.241.0.
+  This version contains a crucial update to address the inability to add new securities, an issue that arose from a recent change to Yahoo Finance.
   A full list of new features, improvements, and bug fixes
-  for versions between 1.231.0 and 1.241.0 can be found here: https://github.com/ghostfolio/ghostfolio/releases.
-    
-  Version 1.241.0 release notes:
+  for versions between 1.241.0 and 1.269.0 can be found here: https://github.com/ghostfolio/ghostfolio/releases.
 
-  - Filtered activities with type ITEM from search results
 
-  - Considered the user's language in the Stripe checkout
+  Version 1.269.0 release notes:
 
-  - Upgraded the Stripe dependencies
+  - Added FINANCIAL_MODELING_PREP as a new data source type
 
-  - Upgraded twitter-api-v2 from version 1.10.3 to 1.14.2
+  - Improved the market price on the first buy date in the chart of the position detail dialog
   
+  - Restructured the admin control panel with a new settings tab
```

Comment:
In order to have each of these show up on their own separate lines in the UI, I added a newline between each.